### PR TITLE
Fix modifier for functions made for governance calls

### PR DESCRIPTION
**Pull request by danoctavian**
## Context

Fixed audit issue: https://github.com/NexusMutual/smart-contracts/issues/323

## Changes proposed in this pull request

Change modifiers for:

[MemberRoles::changeMaxABCount()](https://github.com/NexusMutual/smart-contracts/blob/14a7ed6f5b6df877a2cb7eb9fd390c9cb216ccb2/contracts/modules/governance/MemberRoles.sol#L71)
[TokenController::changeOperator()](https://github.com/NexusMutual/smart-contracts/blob/14a7ed6f5b6df877a2cb7eb9fd390c9cb216ccb2/contracts/modules/token/TokenController.sol#L54)

To be onlyGovernance as opposed to onlyInternal since that is the actual minimal permission necessary.

Include a summary of the change.

## Test plan

Run existing tests and make sure they don't fail.

The fix is quite minimal.

## Checklist

- [x] Rebased the base branch
- [x] Attached corresponding Github issue
- [x] Prefixed the name with the type of change (i.e. feat, chore, test)
- [x] Performed a self-review of my own code
- [x] Followed the style guidelines of this project
- [x] Made corresponding changes to the documentation
- [x] Didn't generate new warnings
- [x] Didn't generate failures on existing tests
- [x] Added tests that prove my fix is effective or that my feature works

## Review

When reviewing a PR, please indicate intention in comments using the following emojis:
* :cake: = Nice to have but not essential.
* :bulb: = Suggestion or a comment based on personal opinion
* :hammer: = I believe this should be changed.
* :thinking: = I don’t understand something, do you mind giving me more context?
* :rocket: = Feedback

### CODE REVIEW

File: test/unit/TokenController/setup.js [hunk 1]
```diff
@@ -32,22 +32,26 @@ async function setup() {
...
   await tokenController.changeDependentContractAddress();
-  await tokenController.connect(internal).changeOperator(tokenController.address);
   await nxm.mint(tokenController.address, ethers.utils.parseUnits('1000'));
 
   for (const member of [...members, tokenController]) {
     await master.enrollMember(member.address, 2);
     await tokenController.connect(internal).addToWhitelist(member.address);
   }
 
+  master.setLatestAddress(hex('GV'), accounts.governanceContracts[0].address);
```
Comment (roxdanila) [inline test/unit/TokenController/setup.js:44]: what's the reason for this change? @danoctavian

Comment (danoctavian) [reply test/unit/TokenController/setup.js:44]: as discussed in DMs, it's just so one can execute the changeOperator call which is now restricted to onlyGovernance

Injecting an EOA as GV temporarily to call that for the test setup

**Review by shark0der — APPROVED** [review]

**Review by kyledewy — APPROVED** [review]
